src/olympia/devhub/templates/devhub/addons/submit/source.html

```diff
@@ -0,0 +1,91 @@
+{% extends "devhub/addons/submit/base.html" %}
+
+{% block title %}{{ dev_page_title(_('Source Code Upload'), addon) }}{% endblock %}
+
+{% block primary %}
+
+{% set channel_param = 'listed' if channel == amo.RELEASE_CHANNEL_LISTED else 'unlisted' %}
+
+<form method="post" enctype="multipart/form-data" id="submit-source">
+    {% csrf_token %}
+    <h3>{{ _('Submit Source Code') }}</h3>
+    <div>
+        {{ form.non_field_errors() }}
+        <p>
+        {% trans %}
+            You may need to submit source code. If you don’t submit the correct
+            source code now, your add-on may be removed from the site!
+        {% endtrans %}
+        </p>
+        <p id="source-code-submission-question">
+        {% trans a_attrs = 'href="https://developer.mozilla.org/en-US/Add-ons/Source_Code_Submission" target="_blank" rel="noopener noreferrer"'|safe %}
```

Comment:
Does that mean the link is translatable? It shouldn't be.

Reply:
No, that's a way of setting a constant that's local to that `{% trans %}` block that can be used inside it.  Only the content inside the block is translatable (from the `%}` to `{% endtrans %}`)

Reply:
Got it, thanks!